solidity/test/bridge/Bridge.test.ts

```diff
@@ -5398,8 +5433,14 @@ describe("Bridge", () => {
                 )
               })
 
-              it("should slash the wallet's operators", async () => {
-                // TODO: Add test when operator slashing is implemented
+              it("should call the ECDSA wallet registry's seize function", async () => {
+                expect(walletRegistry.seize).to.have.been.calledOnceWith(
+                  redemptionTimeoutSlashingAmount,
+                  redemptionTimeoutNotifierRewardMultiplier,
+                  await thirdParty.getAddress(),
+                  data.wallet.ecdsaWalletID,
+                  walletMembersIDs
+                )
```

Comment:
```
describe("notifyRedemptionTimeout", () => {
    context("when redemption request exists", () => {
      context("when the redemption request has timed out", () => {
        context("when the wallet is in Live state", () => {
          context("when the wallet is the active wallet", () => {
            context("when the wallet does not have the main UTXO set", () => {
```

I am confused about it. If the wallet is Live and has no main UTXO set, how could it be requested for redemption? cc @lukasz-zimnoch

Reply:
@pdyraga 
It seems you are right (if the wallet is in the `Live` state and has a pending redemption request, it must have a `mainUTXO` as well).
So the situation in which the `Live` wallet has no `mainUTXO` cannot happen in reality.

However, in unit tests we could do that by modifying the state of the `BridgeStub` (we have `unsetWalletMainUtxo` function).
I added this "when the wallet does not have the main UTXO set" `context`, because In the flow of `notifyRedemptionTimeout` we reuse the `moveFunds` and there is an if statement that checks for presence or absence of `mainUTXO` and I wanted to test it.  
I didn't realise the wallet lifecycle does not allow for this.

So to fix this I suggest we remove `context("when the wallet does not have the main UTXO set"...)`  from the Live state.